app/components/index/ember-addons.hbs

```diff
@@ -3,6 +3,7 @@
     class="addon-tabs--header"
     role="tablist"
   >
+    {{!-- template-lint-disable "no-down-event-binding" --}}
```

Comment:
I have a plan to refactor this component, and will address the event recommendation then.